compiler/rustc_target/src/spec/targets/x86_64_unknown_managarm_mlibc.rs

```diff
@@ -0,0 +1,22 @@
+use crate::spec::{base, Cc, LinkerFlavor, Lld, StackProbeType, Target, TargetMetadata};
+
+pub fn target() -> Target {
+    let mut base = base::managarm_mlibc::opts();
+    base.cpu = "x86-64".to_string().into();
+    base.max_atomic_width = Some(64);
+    base.add_pre_link_args(LinkerFlavor::Gnu(Cc::Yes, Lld::No), &["-m64"]);
+    // don't use probe-stack=inline-asm until rust-lang/rust#83139 is resolved.
```

Comment:
https://github.com/rust-lang/rust/issues/83139 is closed since 2021, should this point to something else? Just outdated?